staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -181,13 +182,27 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		serviceResolver:          c.ExtraConfig.ServiceResolver,
 	}
 
+	apiResourceConfig, err := resourceconfig.MergeAPIResourceConfigs(
```

Comment:
Can we turn this boilerplate code into a shared `APIEnablementOptions` with an `ApplyTo` func? And then in `GenericApiServer.Complete` we can fill in a default "allow all resources" `APIResourceConfigSource`?

Reply:
merged apiResourceConfig currently is used locally, I do not make it shared. And I make APIEnablementOptions into RecommendedOptions.